planning/behavior_velocity_planner/include/scene_module/speed_bump/manager.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2022 LeoDrive.ai, Inc.
```
Please modify other files

Reply:
I didnt create a uniqe module actually. this module looks like crosswalk and stop line modules. I'm not sure it's ethical to make this change.

Reply:
Thank you for reply
I see :+1: